swagger.yaml

```diff
@@ -0,0 +1,244 @@
+swagger: "2.0"
+info:
+  description: "discretionary-business-grants API Schema"
+  version: "0.0.1"
+  title: "discretionary-business-grants API"
+host: "discretionarybusinessgrants.hackney.gov.uk"
+basePath: "/api"
+tags:
+  - name: "urls"
+    description: "Secure URL for uploading files"
```

Comment:
This to me implies that this is a secure url but it's an api for obtaining secure urls.

Reply:
Updated :)

Reply:
Tweaked in: 82ff1223a915564329dfa411f398e170531825ee
WDYT?